cob_image_flip/ros/src/image_flip.cpp

```diff
@@ -292,8 +358,9 @@ void ImageFlip::imageCallback(const sensor_msgs::ImageConstPtr& color_image_msg)
 		rot_mat = cv::getRotationMatrix2D(center, -rotation_angle, 1.0);
 		if (switch_aspect_ratio==true)
 		{
-			rot_mat.at<double>(0,2) += 0.5*(color_image_turned.cols-color_image.cols);
```

Comment:
Just leave it the old way in this case, because the rotation matrix is always the same format CV_64FC1.